npm/package.js

```diff
@@ -0,0 +1,97 @@
+var shell = require('shelljs'),
+  path = require('path'),
+  async = require('async'),
+  fs = require('fs'),
+  exists,
+  codegen,
+  codegen_path,
+  getSubfolders,
+  individual_test,
+  commandOut,
+  pwd = shell.pwd();
+
+  const args = process.argv[2],
+  PATH_TO_CODEGENS_FOLDER = path.resolve(__dirname, '../codegens');
+
+// throw JS error when any shell.js command encounters an error
+shell.config.fatal=true;
+
+// ensure that the working tree is clean before packaging
+commandOut = shell.exec('source ./npm/package/require_clean_work_tree.sh');
+if (commandOut.code !== 0) {
+    shell.exit(1);
+}
+/* commandOut = shell.exec('require_clean_work_tree "create package"');
+if (commandOut.code !== 0) {
+    console.log('Working tree is not clean, please commit/stash your changes');
+    shell.exit(1);
+} */
```

Comment:
You can remove this, no need to keep commented code.

Reply:
Done